roles/common/tasks/facts.yml

```diff
@@ -5,10 +5,8 @@
     register: CA_password
 
   - name: Generate p12 export password
-    shell: >
-      openssl rand 8 |
-      python -c 'import sys,string; chars=string.ascii_letters + string.digits + "_@"; print("".join([chars[ord(c) % 64] for c in list(sys.stdin.read())]))'
-    register: p12_password_generated
+    set_fact:
+      p12_password_generated: "{{ lookup('password', '/dev/null length=9 chars=ascii_letters,digits,!,@,#,$,%,^,&,*,<,>,/,?') }}"
```

Comment:
This way won't work. There is a fact for p12 https://github.com/trailofbits/algo/blob/eb61e393726b279a70d31b2ff6fbbd1d56c2fe16/roles/common/tasks/facts.yml#L15-L18